servers/physics/body_pair_sw.cpp

```diff
@@ -372,17 +372,35 @@ void BodyPairSW::solve(real_t p_step) {
 
 			Vector3 jb = c.normal * (c.acc_bias_impulse - jbnOld);
 
-			A->apply_bias_impulse(c.rA + A->get_center_of_mass(), -jb);
-			B->apply_bias_impulse(c.rB + B->get_center_of_mass(), jb);
+			A->apply_bias_impulse(c.rA + A->get_center_of_mass(), -jb, MAX_BIAS_ROTATION / p_step);
+			B->apply_bias_impulse(c.rB + B->get_center_of_mass(), jb, MAX_BIAS_ROTATION / p_step);
 
-			c.active = true;
+			crbA = A->get_biased_angular_velocity().cross( c.rA );
+			crbB = B->get_biased_angular_velocity().cross( c.rB );
+			dbv = B->get_biased_linear_velocity() + crbB - A->get_biased_linear_velocity() - crbA;
+
+			vbn = dbv.dot(c.normal);
+
+			if (Math::abs(-vbn + c.bias) > MIN_VELOCITY) {
```

Comment:
If limiting the angular velocity change caused the velocity in the collision point to not be the desired one, then we apply a second impulse at the center of mass to add the missing velocity without adding further angular velocity